packages/loader/driver-utils/src/treeConversions.ts

```diff
@@ -52,7 +52,7 @@ export function convertSummaryTreeToSnapshotITree(
                     parsedContent = Uint8ArrayToString(value.content, "base64");
                     encoding = "base64";
                 }
-                entries.push(new BlobTreeEntry(k, parsedContent, encoding));
+                entries.push(new BlobTreeEntry(k, parsedContent, encoding) as ITreeEntry);
```

Comment:
Similar question - why do we need a cast?

Reply:
as before, I just reverted it - it was hte protocol-base mismatch.